contents/blog/cheaper-analytics.md

```diff
@@ -0,0 +1,66 @@
+---
+date: 2024-03-21
+title: 'PostHog is now up to 12x cheaper'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+category: PostHog news
+tags:
+  - Product updates
+---
+
+## tl;dr
+
+* It is now possible to use PostHog's product and web analytics for up to ~12x less, depending on your volumes
+* This change applies to all existing and new customers for any events that have no person properties set
```

Comment:
This isn't precisely true. You can still send an event with no new person properties, but get charged the "events with person properties" price.

```suggestion
* This change applies to all existing and new customers for any events where we don't do person property processing
```